### PR TITLE
Easier iOS plugin integration

### PR DESCRIPTION
**Pull request by Sergio-Mira**

### CODE REVIEW

**Comment by seaders** [comment]
I don't think we should be symlinking like this, here in the public side.  We can package them in the .unityPackage (and copy them in during CI), but there should just be a README file here, and in Assets/Plugins/iOS that says if you're working from github, you need to go grab these folders from the released zip file, or from swrve-ios-sdk.

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,6 @@
 /docs/
 /buildtemp/*
 *.log
-.DS_Store
\ No newline at end of file
+/unity3d/Assets/Plugins/iOS/SwrveConversationSDK*
```
Comment (Sergio-Mira) [inline .gitignore:5]: 👍

**Comment by Sergio-Mira** [comment]
Yeah I agree as I mentioned in the other pr 💃

**Comment by seaders** [comment]
So with that, we good to 🚀 ?  I'm happy with the state, and it seems like she's working as expected.

🚀

**Pull request merged**